### PR TITLE
Migrate old blog posts from farmier.com

### PR DESCRIPTION
**Pull request by mstenta**
I have two old blog posts from 2019 on farmier.com that I think would make sense moving to farmOS.org for historic preservation. I brought this up on a community call a while back and there seemed to be agreement that it made sense, but I didn't get around to it until now.

One is about the release of farmOS v1 and the other is about how farmOS development is funded. Neither is specific to Farmier, and Farmier is only mentioned in a brief paragraph in the "Funding farmOS" post to explain the relationship to farmOS.

You can view them on farmier.com here:

- https://farmier.com/post/2019/farmOS-1.0/
- https://farmier.com/post/2019/funding-farmOS/

If we merge these into the farmOS.org blog, I will delete them from farmier.com.

### CODE REVIEW

File: posts/2019/farmOS-1.0/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: farmOS 1.0
+date: 2019-03-20
+author: Michael Stenta
+slug: 2019/farmOS-1.0
+---
+
+# farmOS 2.0
```
Comment (wotnak) [inline posts/2019/farmOS-1.0/index.md]:
```suggestion
# farmOS 1.0
```
image

Comment (mstenta) [reply posts/2019/farmOS-1.0/index.md]: Good catch @wotnak! My fingers only know how to type "2.0" now. :-)

I'll push a fixup commit...

Comment (mstenta) [reply posts/2019/farmOS-1.0/index.md]: https://github.com/farmOS/farmOS-community-blog/pull/17/commits/66593522a2ed38f84536343675c6529ec49fcb7d

**Comment by mstenta** [comment]
I just did a quick click through and found two broken links... so I will fix those in a fixup commit...

**Comment by mstenta** [comment]
Bumping this back up and requesting review from a few folks. I'd like to remove the blog from farmier.com, but don't want to lose these.

Any objections to moving them over?

**Review by wotnak — APPROVED** [review]

**Comment by mstenta** [comment]
Merged my fixup commits, rebased onto `main`, and force-pushed (no changes).

**Review by paul121 — APPROVED** [review]

**Pull request merged**